src/storage/CookiesStorage.js

```diff
@@ -67,21 +67,67 @@ class CookiesStorage {
     if (document.cookie === oldCookies) {
       return null;
     }
-    
+
     // Since the cookie was updated with a dummy value, remove it by setting
     // expiry to a date in the past.
-    const pastDate = "Thu, 01 Jan 1970 00:00:00 GMT";
-    document.cookie = 
+    const pastDate = 'Thu, 01 Jan 1970 00:00:00 GMT';
+    document.cookie =
       `registrable_domain=;domain=${domain};expires=${pastDate}`;
 
     return domain;
   }
 
   /** @override */
-  save(key, value, secondsToLive) {}
+  save(key, value, secondsToLive =
+    /** @type {number} */ (this.settings_['expires'])) {
```

Comment:
Is it intentional that the guideline isn't followed here?
https://docs.google.com/document/d/1XIhihp4elYAa9cFqJL9vg3f7u4kbsoZqIQa5ytBqYg0/edit#heading=h.x95stkrs2rc2